PlayniteSounds.cs

```diff
@@ -6,9 +6,7 @@
 using System.IO;
```

Comment:
It may be beneficial to just to look at the raw file, the diff is too overwhelming.